app/release/output.json

```diff
@@ -0,0 +1 @@
+[{"outputType":{"type":"APK"},"apkInfo":{"type":"MAIN","splits":[],"versionCode":141,"versionName":"6.2.1","enabled":true,"outputFile":"calendar-release.apk","fullName":"release","baseName":"release"},"path":"calendar-release.apk","properties":{}}]
```

Comment:
remove these 3 files from the pull request

Reply:
Just finished deleting them. Any ideas on why the /release folder was present?

Reply:
not sure how come it doesnt show to me either actually, since it isnt added to .gitignore. Anyway before doing a commit always check what files are modified. Looks good now, thanks.